apps/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -95,13 +95,28 @@ export class OperationExecutionManager {
       if (parallelism === 'max') {
         this._parallelism = numberOfCores;
       } else {
-        const parallelismInt: number = parseInt(parallelism, 10);
+        const parallelismAsNumber: number = Number(parallelism);
 
-        if (isNaN(parallelismInt)) {
-          throw new Error(`Invalid parallelism value of '${parallelism}', expected a number or 'max'`);
+        const getParallelismValueHighThanZero = (value: number) => (value > 0 ? value : 1);
+
+        if (typeof parallelism === 'string' && parallelism.trim().endsWith('%')) {
+          const parsedPercentage = Number(parallelism.trim().replace(/\%$/, ''));
+
+          if (parsedPercentage <= 0 || parsedPercentage > 100) {
```

Comment:
Do we actually need to cap it at number of cores? If you specify an integer we allow you to set as many parallel processes as you want, which may be relevant if you have a lot of packages that spend a lot of time waiting rather than using the CPU.

Reply:
Let's tackle that separately if someone requests it.